src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/AsyncCollectorArgumentBindingProvider.cs

```diff
@@ -1,84 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
```

Comment:
Absolutely beautiful to see all these types removed. Are you **sure** the base helpers are handling all these scenarios? Seems too good to be true. I guess we have to rely on the existing tests, but lets make sure you've eyeballed all these deleted classes to look for any quirky things that might not have tests (maybe you already did this).